task/utils.go

```diff
@@ -0,0 +1,32 @@
+package task
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"k8s.io/client-go/rest"
+)
+
+const (
+	QPSRate   = "QPS_RATE"
+	BurstRate = "BURST_RATE"
+)
+
+func SetRateLimiter(config *rest.Config) error {
```

Comment:
It doesn't seem right to put this in the `task` package as it is unrelated to k8s. We can use the `k8s/common` package.

Reply:
moved to k8s/common